### PR TITLE
Set up windows CI

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Test against this version of Node.js
+environment:
+  matrix:
+  # node.js
```
Comment (ofrobots) [inline appveyor.yml]: Add a note on why 0.12 is not being tested.

Comment (matthewloring) [inline appveyor.yml]: Added.

**Comment by matthewloring** [comment]
@ofrobots Ready for another look.

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**